app/assets/javascripts/ui_components/toolbar.coffee

```diff
@@ -0,0 +1,10 @@
+$ ->
+  # all checked checkboxes display as active
+  $(".toolbar .checkbox input[type='checkbox']:checked").parents('.checkbox').addClass('active')
```

Comment:
@dziemba unfortunately not, because bootstrap nests the checkbox within the label, instead of having it adjacent. so it is the parent that needs to be targeted based on the state of the child :( this isn't possible with css alone.

the alternative would be changing the checkbox-list component to format the checkbox-lists in a non-bootstrap way. which seems likely to cause more issues elsewhere or in the future, depending on where/how the component is used.

so i figured since the toolbar is more the exception case with displaying a list of checkboxes, it was more appropriate to let it have a magical implementation, and keep the component standard bootstrap.